dlme_airflow/utils/requests_session.py

```diff
@@ -0,0 +1,22 @@
+import requests
```

Comment:
Request timeout resilience is something that many tasks will need. I wasn't sure where to put this or if its already defined somewhere else so added this in utils. Let me know if there is a better place to put this. Trying to avoid unnecessary repetition.